specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -247,9 +239,7 @@ def add_polynomialcoeff(a: PolynomialCoeff, b: PolynomialCoeff) -> PolynomialCoe
     Sum the coefficient form polynomials ``a`` and ``b``.
     """
     a, b = (a, b) if len(a) >= len(b) else (b, a)
-    length_a = len(a)
-    length_b = len(b)
-    return [(a[i] + (b[i] if i < length_b else 0)) % BLS_MODULUS for i in range(length_a)]
+    return PolynomialCoeff([a[i] + (b[i] if i < len(b) else BLSFieldElement(0)) for i in range(len(a))])
```

Comment:
If I remember well, this code was using the utility variables `length_a` and `length_b` because hww found that `len()` appeared too much on the profiler (because `len` is called at every `for` loop iteration even tho its value doesn't change).

Reply:
Good call. I forgot that.